scripts/populate.mjs

```diff
@@ -1,131 +1,194 @@
+    text: "The pizza shop down the street is giving out free donuts. Kinda sketchy"
   },
   {
     address: "0x1cbd3b2770909d4e10f157cabc84c7264073c9ec",
     id: "0x1000e",
     pk: "0x47c99abed3324a2707c28affff1267e45918ec8c3f20b8aa892e8b065d2942dd",
     name: "Juan Harrington",
+    attachment: createAudioAttachment("https://archive.org/download/78_the-one-i-love-belongs-to-somebody-else_ella-fitzgerald-albert-von-tilzer-lew-bro_gbia8001485/01%20-%20I%27M%20THE%20LONESOMEST%20GAL%20IN%20TOWN%20-%20ELLA%20FITZGERALD.mp3",
+      "audio/mpeg"),
+    text: "An MP3 featuring my fave, Ella Fitzgerald!"
   },
   {
     address: "0xdf3e18d64bc6a983f673ab319ccae4f1a57c7097",
     id: "0x1000f",
     pk: "0xc526ee95bf44d8fc405a158bb884d9d1238d99f0612e9f33d006bb0789009aaa",
-    name: "Axel Howlett",
+    name: "Srikanth Raghavan",
+    attachment: createAudioAttachment("https://stream.thisamericanlife.org/629/P41zu7fBNaTorAhTWbSHpLTmRj2MvvEdKIOiNBT2xOA/629.mp3", "audio/mpeg"),
+    text: "Expect delays.  (MP3)"
   },
   {
     address: "0xcd3b766ccdd6ae721141f452c550ca635964ce71",
     id: "0x10010",
     pk: "0x8166f546bab6da521a8369cab06c5d2b9e46670292d85c875ee9ec20e84ffb61",
-    name: "Jonathan Weakman",
+    name: "John Mando",
+    attachment: createImageAttachment("http://www.placekitten.com/1024/768"),
+    text: "This is the way."
   },
   {
     address: "0x2546bcd3c84621e976d8185a91a922ae77ecec30",
     id: "0x10011",
     pk: "0xea6c44ac03bff858b476bba40716402b03e41b8e97e276d1baec7c37d42484a0",
-    name: "Percival Lite",
+    name: "Eskil Grímsdóttir",
+    attachment: createAudioAttachment("https://archive.org/download/penguin_island_ms_librivox/penguin_island_01_france.ogg", "audio/ogg"),
+    text: "Penguin Island (OGG-Vorbis)"
   },
   {
     address: "0xbda5747bfd65f08deb54cb465eb87d40e51b197e",
     id: "0x10012",
     pk: "0x689af8efa8c651a91ad287602527f3af2fe9f6501a7ac4b061667b5a93e037fd",
     name: "Isaiah Brownhill",
+    attachment: createImageAttachment("https://upload.wikimedia.org/wikipedia/commons/2/20/Baby_2.jpg"),
+    text: "My favorite cartoon is Spongebob"
   },
   {
     address: "0xdd2fd4581271e230360230f9337d5c0430bf44c0",
     id: "0x10013",
     pk: "0xde9be858da4a475276426320d5e9262ecfc3ba460bfac56360bfa6c4c28b4ee0",
     name: "Howard Fergusson",
+    attachment: createImageAttachment("https://live.staticflickr.com/6064/6031231222_eea9093801_b.jpg"),
+    text: "`Let your plans be dark and impenetrable as night, and when you move, fall like a thunderbolt.`\n" +
+      "―Sun Tzu, The Art of War"
   },
   {
     address: "0x8626f6940e2eb28930efb4cef49b2d1f2c9c1199",
     id: "0x10014",
     pk: "0xdf57089febbacf7ba0bc227dafbffa9fc08a93fdc68e1e42411a14efcf23656e",
-    name: "Alan Dart",
+    name: "Davonte Franklin, Jr.",
+    attachment: createImageAttachment("https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT1B8ZsbUPkarzs2ZsMTpAoICuqPiFrEIrKyQ&usqp=CAU"),
```

Comment:
So excited to have these attachments in the script!  One thing that I am not seeing though is a post that has multiple attachments.  Might be nice to cover the following cases:
- no attachment
- one image attachment
- one video attachment
- one audio attachment
- a post with multiple attachments of the same type
- a post with multiple attachments of different types

Reply:
Hm, well I was going to save it for another PR but I bet it won't take long.